Client/Frontend/Brave Notifications/Brave Wallet/WalletConnectionView.swift

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import Shared
+
+class WalletConnectionView: UIControl {
+  private let stackView: UIStackView = {
+    let result = UIStackView()
+    result.axis = .horizontal
+    result.spacing = 15
+    result.alignment = .center
+    result.isUserInteractionEnabled = false
+    return result
+  }()
+
+  private let iconImageView: UIImageView = {
+    let result = UIImageView(image: UIImage(imageLiteralResourceName: "brave.unlock"))
+    result.tintColor = .white
+    return result
+  }()
+
+  private let titleLabel: UILabel = {
+    let result = UILabel()
+    result.textColor = .white
+    result.font = .systemFont(ofSize: 15, weight: .semibold)
```

Comment:
Usually I would say no hardcoded fonts, but given its a notification we may not want to grow it too large.

If you're going to use hardcoded fonts, ensure you add a `UILargeContentViewerInteraction` to the view and that you lock in `brave.unlock` to the specific font size you want otherwise it will grow. 

If you're not then use `subheadline` font style and apply semibold to it